mlxtend/plotting/corr_circle_pca.py

```diff
@@ -0,0 +1,97 @@
+# mlxtend Machine Learning Library Extensions
+#
+# A function for plotting a PCA correlation circle
+# File Author: Gabriel Azevedo Ferreira <az.fe.gabriel@gmail.com>
+
+from sklearn.decomposition import PCA
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from adjustText import adjust_text
+
+
+def create_correlation_table(A, B, names_cols_A, names_cols_B):
```

Comment:
Could you use the NumPy docstring convention for docstrings? Otherwise, the documentation will not be created properly (the documentation building tools rely on that particular docstring format). See the other docstrings for examples how they should look like.